Problem schema error on lambda

- Fix broken schema
``` python
[ERROR] TypeError: __new__() got an unexpected keyword argument 'type'
Traceback (most recent call last):
  File "/var/task/app.py", line 107, in handler
    msg = parse_message_payload(event)
  File "/var/task/app.py", line 83, in parse_message_payload
    return Payload(**body)
```

--- a/app.py
+++ b/app.py
@@ -80,7 +80,12 @@ def parse_message_payload(event: dict) -> Payload:
     if missing_fields:
         raise PayloadMissingFieldsError(missing_fields)
 
-    return Payload(**body)
+    return Payload(
+        to=body['to'],
+        subject=body['subject'],
+        html=body['html'],
+        text=body['text'],
+    )
 
 
 def send_email(cfg: Config, msg: Payload) -> None:

